### DISCUSSION TITLE
How to open wrapper settings?

### DISCUSSION THREAD

**Discussion by Crazor** [The-Wineskin-Project/WineskinServer#214] [Q&A] [answered]
It used to be the case that you could re-open the settings dialog of the wrapper after an executable has been set to be run on launch by holding the CMD key (or was it alt?). However, with this fork, I cannot make this work anymore. Has the combination changed? Or am I remembering this wrong? I cannot find any hint in the docs...

**Comment by LewsTherinTelescope** [comment, marked as answer]
Not sure if there's any shortcut or not, but right-click > `Show Package Contents` > `Wineskin.app` works.
